hack/run.sh

```diff
@@ -96,6 +95,8 @@ elif [[ "${action}" == "benchmark-filter" ]]; then
   "${ROOT_DIR}/data-plane/benchmarks/run.sh" "$2"
 elif [[ "${action}" == "benchmark-filters" ]]; then
   "${ROOT_DIR}/data-plane/benchmarks/run.sh"
+elif [[ "${action}" == "./hack/update-codegen.sh" || "${action}" == "./hack/run.sh format-java" ]]; then
+    "${ROOT_DIR}"/mvnw spotless:apply
```

Comment:
@MeenuyD sorry for the confusion from my original comment on the issue - I think your code needs to:
1. `cd` to `${ROOT_DIR}/data-plane`
2. Run `./mvnw spotless:apply`